### ISSUE TITLE
No print preview on printer display

### ISSUE THREAD

**Issue by FrankHovis** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#30]
Has anyone managed to work out what format/position/magic is needed to get a print preview on the printer's display?

**Comment by suchmememanyskill** [comment]
What do you mean with preview?

If you mean the image, as far as i know no. I have also never seen it work, does the screen even have this functionality?

**Comment by FrankHovis** [comment]
Yes the image preview on the screen.  It works on similar Ender models like the V2 with the use of a post processing script (https://github.com/KenHuffman/UltimakerCuraScripts) but nothing I've tried has yet worked. 

I'd be surprised if the screen doesn't have th... what am I saying - this is Creality, I **wouldn't** be surprised if the screen doesn't have the functionality, but it seems odd it doesn't.

**Comment by potatopaste** [comment]
Print preview is not supported in Ender 3 V3 SE

**Comment by FrankHovis** [comment]
> Print preview is not supported in Ender 3 V3 SE

Says who?

**Comment by boecko** [comment]
Says Creality
https://www.reddit.com/r/Ender3V3SE/comments/16o7oj9/how_to_generate_correct_thumbnail/

 
> Contacted creality support and answer from their tech team was:
> "Hi, the screen of this printer itself doesn't support preview image, it can't realize what you wa.
> Which is kinda crazy since the default image is there but well... That's how it goes I guess.
>
